src/core/amp-globals.d.ts

```diff
@@ -1,16 +1,25 @@
 export {};
```

Comment:
fine with whatever you decide, but my impression was you wanted to name this `globals.d.ts`

Reply:
resolved offline